app/views/layouts/application.html.erb

```diff
@@ -13,20 +13,42 @@
 
   <body>
     <% unless current_page?(sign_in_path) %>
-      <nav class="crumbs">
-          <ul>
-              <li class="crumb"><a href="#"><%= t(".title") %></a></li>
-              <li class="crumb"><%= link_to t(".sign_in"), sign_in_path %></li>
-          </ul>
-      </nav>
+      <div class="header sticky top-0 bg-white shadow-md flex
+                    items-center px-8 py-02 h-16 flow-root">
+        <div class="float-left mt-3">
+          <a class="p-2 text-3xl" href="#">
+            <%= t(".title") %>
+          </a>
+        </div>
```

Comment:
In a later change, it would be a good idea to change this to a `#link_to`. It should also point at the `root_path`. Add a card to our Trello board, and we can do it later.

Reply:
Trello card added: 
[Trello card](https://trello.com/c/sSIcE48I)